packages/blockchain-wallet-v4-frontend/src/scenes/Settings/Addresses/Btc/ManageAddresses/UnusedAddresses/index.js

```diff
@@ -44,6 +44,7 @@ const mapStateToProps = (state, props) => {
   const isDefault = parseInt(props.walletIndex) === Types.HDWallet.selectDefaultAccountIdx(Types.Wallet.selectHdWallets(state.walletPath.wallet).get(0))
   const account = Types.Wallet.selectHDAccounts(state.walletPath.wallet).get(props.walletIndex)
   const labels = Types.HDAccount.selectAddressLabels(account).reverse().toArray()
+  // TODO: remove addresses with tx > 0
```

Comment:
what does this mean?

Reply:
there are multiple bugs with the unused addresses code. just added that tag there so i remember to fix them later today.